skimage/graph/__init__.py

```diff
@@ -1,3 +1,7 @@
+"""
+Utilities for Image Segmentation, Object detection and Network Analysis.
```

Comment:
Could you maybe highlight that this module is focused on "graph-based" segmentation and object detection to make it more distinct from other modules such as `skimage.segmentation`?

Reply:
Thank You @lagru for this suggestion. Will it be good to modify it as "Utilities for Graph-based Image Segmentation, Object Detection, and Network Analysis"?